azure/functions/timer.py

```diff
@@ -1,23 +1,30 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
```

Comment:
Why is this removed?

Reply:
It was mistakenly removed. I have added in new commit.